Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,93 @@ class Program
     {
         static void Main(string[] args)
         {
+            PrintBMI();
+            PrintBMI();
+        }
+
+        static string AskName(string nameInput)
+        {
+            bool isInvalidName;
+            Console.WriteLine(nameInput);
+            string name = Console.ReadLine();
+            isInvalidName = string.IsNullOrEmpty(name);
+            if (isInvalidName)
+            {
+                Console.WriteLine("Name cannot be empty.");
+                return "-";
+            }
+                return name;
+        }
+
+        static int AskAge(string agePrompt)
+        {
+            bool isValidAge;
+            Console.WriteLine(agePrompt);
+            string ageString = Console.ReadLine();
+            isValidAge = string.IsNullOrEmpty(ageString); 
+            if (isValidAge) return 0; 
```

Comment:
This whole function is a blob. Please separate parts of function with a blank line to better clarify distinct parts of a function.